ebs-pin

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
```

Comment:
Firstly, thanks for the PR.

Is the python3 change necessary?  
Could it just be changed to `/usr/bin/env python` and the current version is used?

It's been a while since I looked at this

Reply:
The changes I made have probably broken backwards compatibility with 2.7. It's possible to make it backwards compatible (http://python-future.org/automatic_conversion.html ?), but I don't see much point as Python 2 is EOL soon (https://pythonclock.org/).

Reply:
Just because it is EOL doesn't mean people will stop using it, but I'll merge it regardless and I'll create a PyPi package with a different name for for python3